### DISCUSSION TITLE
Selection not cleared when navigation changed

### DISCUSSION THREAD

**Discussion by licitdev** [directus/directus#8464] [Archive]
May I ask if retaining the selection despite navigation changed in File Library the intended behaviour?
Or should it be cleared upon a navigation change?

It occurs in User Directory too.

https://user-images.githubusercontent.com/26413686/135486259-9e1bcbc0-b6c8-45b6-8dd1-bfdb44992cc5.mov

**Comment by rijkvanzanten** [comment]
It should be cleared on navigation 👍🏻

**Comment by licitdev** [comment]
Pushed fix at #8472.
